### PR TITLE
UI of calculator Component

### PR DESCRIPTION
**Pull request by aron-net**
- Development of the UI of the calculator.
- Class-based components.
- Live dome is in progress...

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,42 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Button from './Calculator/Button';
+import Screen from './Calculator/Screen';
+import ButtonCard from './UI/ButtonCard';
+/* eslint-disable react/destructuring-assignment */
```
Comment (alexander16108) [inline src/App.js]: as per the requirements you are required to import the react components as so `import React, { Component } from 'react';`

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @aron-helu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
just a tiny bit left

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Calculator/Button.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import './Button.css';
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+
+const Button = ({ className, value }) => (
+  <button className={className}>{value}</button>
+);
+
+export default Button;
```
Comment (bimbolabuari) [inline src/Calculator/Button.js]: [Optional] As part of the project requirement, Kindly create a directory called `components`, then you can move your `Calculator` folder with the calculator files inside the components directory. 👍 


> image

**Review by bimbolabuari — COMMENTED** [review]
Hello @aron-helu  👋,

_This is an additional review as a result of the second opinion request._

- While the previous CR was following the code review guideline, I think the CR omitted the fact that you extend your `component` from `React` using `React.Component`, if you had extend just `Component` from react then the requested changes would have been applicable.

As this was the only reason your project was not approved it will be approved as soon as possible.

- Kindly implement the optional change in the inline comment as this was a requirement and was omitted by the previous CR. 

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve :)

Keep rocking 🚀!

**Review by nicupop729 — APPROVED** [review]
Hi @aron-helu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 - [X] Correct git flow practice :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**